implementations/rust/ockam/ockam_command/Cargo.toml

```diff
@@ -64,7 +64,7 @@ colorful = "0.2"
 colors-transform = "0.2.11"
 console = "0.15.7"
 ctrlc = { version = "3.4.1", features = ["termination"] }
-dialoguer = "0.10"
+dialoguer = "0.11"
```

Comment:
While we are at it, could you please specify the full version here (`0.11.0`)? That way `dependabot` will be able to upgrade to any new patch version, instead of waiting until the next minor.